README.md

```diff
@@ -87,12 +34,3 @@ General instructions on _how_ to contribute should be stated with a link to [CON
 1. [TERMS](TERMS.md)
 2. [LICENSE](LICENSE)
 3. [CFPB Source Code Policy](https://github.com/cfpb/source-code-policy/)
-
-
-----
-
-## Credits and references
```

Comment:
A lot of what's described in the values comes from the Agile manifesto.  It should be cited here.